.github/workflows/tests.yaml

```diff
@@ -46,3 +47,16 @@ jobs:
 
       - name: Check resources
         run: kubectl get all -n checkpoint-restore-operator-system
+
+      - name: Prepare Checkpoints Directory in Kind Node
+        run: |
+          sudo mkdir -p /var/lib/kubelet/checkpoints
+          sudo chmod 777 /var/lib/kubelet/checkpoints
```

Comment:
We intentionally use restrictive permissions (`700`) for `/var/lib/kubelet/checkpoints` because the checkpoint files may contain sensitive data.